### PR TITLE
Added data-testids for functional tests for AB#13226

### PR DESCRIPTION
**Pull request by rayking29**
Implements [AB#13226](https://dev.azure.com/qslvic/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/13226)

## Description

Added data-testids to page and components to run functional tests against dev for Admin Communication page.

## Testing

- [ ] Unit Tests Updated
- [ ] Functional Tests Updated
- [x] Not Required

## Items to Review:

-   [General PR Guidelines](https://github.com/bcgov/healthgateway/wiki/PRguidance)

### CODE REVIEW

**Review by sslaws — DISMISSED** [review]

File: Apps/Admin/Client/Components/BannerDialog.razor [hunk 1]
```diff
@@ -9,7 +9,7 @@
         <HgBannerFeedback Severity="@Severity.Error" IsEnabled="@(HasAddError || HasUpdateError)" TResetAction="@CommunicationsActions.ResetStateAction">
             @ErrorMessage
         </HgBannerFeedback>
-        <MudForm @ref="Form">
+        <MudForm @ref="Form" data-testid=banner-dialog-modal-text>
```
Comment (MikeLyttle) [inline Apps/Admin/Client/Components/BannerDialog.razor:12]: Quotes are missing from the value here.

File: Apps/Admin/Client/Components/CommunicationsTable.razor [hunk 2]
```diff
@@ -36,26 +36,29 @@
...
+        <MudTd data-testid="comm-table-effective-date" DataLabel="Effective On">@context.EffectiveDate</MudTd>
+        <MudTd data-testid="comm-table-expiry-date" DataLabel="Expires On">@context.ExpiryDate</MudTd>
         <MudTd DataLabel="Actions" Style="text-align:right">
             <div>
                 <MudTooltip Text="@(context.IsExpanded ? "Collapse" : "Expand")">
                     <MudIconButton OnClick="@(() => ToggleExpandRow(context.Id))"
                                    Icon="@(context.IsExpanded ? Icons.Material.Filled.ExpandLess : Icons.Material.Filled.ExpandMore)"
-                                   Color="@Color.Primary" />
+                                   Color="@Color.Primary"
+                                    data-testid="comm-table-expand-btn"/>
```
Comment (MikeLyttle) [inline Apps/Admin/Client/Components/CommunicationsTable.razor:49]: Nit: extra whitespace.

File: Apps/Admin/Client/Components/BannerDialog.razor (hunk 1 above)
Comment (rayking29) [reply Apps/Admin/Client/Components/BannerDialog.razor]: Updated and pushed.

File: Apps/Admin/Client/Components/CommunicationsTable.razor (hunk 2 above)
Comment (rayking29) [reply Apps/Admin/Client/Components/CommunicationsTable.razor]: Updated and pushed.

**Review by BrianMaki — APPROVED** [review]

**Review by MikeLyttle — APPROVED** [review]

**Pull request merged**